doc/manuals/orion-api.md

```diff
@@ -1922,9 +2012,15 @@ This operation returns a JSON object with information about the type:
 
 _**Request query parameters**_
 
-| Parameter    | Optional | Type   | Description  | Example |
-|--------------|----------|--------|--------------|---------|
-| `entityType` |          | string | Entity Type. | `Room`  |
+| Parameter    | Optional | Type   | Description         | Example |
+|--------------|----------|--------|---------------------|---------|
+| `entityType` |          | string | Entity Type.        | `Room`  |
+
```

Comment:
Note that entityType is not actually a query parameter, but an URL parameter, e.g.: `GET /v2/types/T1?options=noAttrDetail`)

```suggestion
| `options` | ✓        | string | Options dictionary.                        | `count` |
```